polygon-zkevm-adaptor/src/query_service.rs

```diff
@@ -77,21 +77,14 @@ pub async fn serve(opt: &Options) {
         .unwrap()
         .get("blockheight", |_, state| {
             async move {
-                let height: usize = state
-                    .hotshot
-                    .get("status/latest_block_height")
-                    .send()
-                    .await?;
+                let height: usize = state.hotshot.get("status/block-height").send().await?;
```

Comment:
Is it better to use `espresso-sequencer-go`?

Reply:
Yes but this predates that by a lot and is basically legacy code at this point IMO.